chore: v9.8.0 (#1807)
jbsparrow/CyberDropDownloader: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [9.8.0] - 2026-05-11

### Added

> - New `--dump-responses` option to save non binary responses (text/HTML/json) to disk

Author: ```suggestion
- New `--dump-responses` option to save non binary responses (text/HTML/JSON) to disk
```

### Changed

- Log downloads requests/responses
- Log flaresolverr requests/responses

> - Prevent requests from bursting over the rate limit when CDL starts

Author: ```suggestion
- Prevent requests bursting at the start of a run
```

### Deprecated

- `--save-pages-html` will be removed on the next major release

### Removed

- Nekohouse support (No longer exists)

### Fixed

- Parsing of json responses from non Selenium forks of Flaresolverr
- Flaresolverr session destruction always failing

> - Handle posts with deleted images (Tiktok)

Author: ```suggestion
- Handle posts with deleted images (TikTok)
```

> - A single video scrape error stoping the entire scrape queue (LeakedZone/Hotleak)

Author: ```suggestion
- A single video download error canceling the entire scrape queue (LeakedZone/Hotleak)
```
